src/Components/Login/signup.js

```diff
@@ -0,0 +1,162 @@
+                    errors.email = "Email is invalid.";
+                    break;
+                case "auth/operation-not-allowed":
+                    errors.password =
+                        "Email/password accounts are not enabled.";
+                    break;
+                case "auth/weak-password":
+                    errors.password = "Please enter a stronger password.";
+                    break;
+                default:
+                    errors.password = "An unexpected error has occurred.";
+                    break;
+            }
+            setAllErrors(errors);
+        }
+    }
+
+    return (
+        <Container component="main" maxWidth="xs">
+            <CssBaseline />
+            <div className={classes.paper}>
+                <Avatar className={classes.avatar}>
+                    <AccountCircle />
+                </Avatar>
+                <Typography component="h1" variant="h5">
+                    Signup
+                </Typography>
+                <form
+                    className={classes.form}
+                    noValidate
+                    onSubmit={createAccount}
+                >
+                    <TextField
+                        variant="outlined"
+                        margin="normal"
+                        required
+                        fullWidth
+                        id="displayName"
+                        label="Display Name"
+                        name="display"
+                        autoComplete="display-name"
+                        autoFocus
+                        onChange={(e) => setDisplayName(e.target.value)}
+                        error={!empty(allErrors.displayName)}
+                        helperText={allErrors.displayName}
+                    />
+                    <TextField
+                        variant="outlined"
+                        margin="normal"
+                        required
+                        fullWidth
+                        id="email"
+                        label="Email Address"
+                        name="email"
+                        autoComplete="email"
+                        autoFocus
+                        onChange={(e) => setEmail(e.target.value)}
+                        error={!empty(allErrors.email)}
+                        helperText={allErrors.email}
+                    />
+                    <TextField
+                        variant="outlined"
+                        margin="normal"
+                        required
+                        fullWidth
+                        name="password"
+                        label="Password"
+                        type="password"
+                        id="password"
+                        autoComplete="current-password"
+                        onChange={(e) => setPassword(e.target.value)}
+                        error={!empty(allErrors.login)}
+                        helperText={allErrors.login}
```

Comment:
Should be allErrors.password

Reply:
Thanks for catching this, after fixing it to allErrors.password it now provides a more helpful message when for example a weak password is given.